### ISSUE TITLE
rest-gen: Templates generates invalid path to js, css files

### ISSUE THREAD

**Issue by arowM** [silkapp/rest#95]
I tried as follows, and got `index.html`s that have invalid paths to js, css.
``` bash
.cabal-sandbox/bin/rest-example-gen -d "./" -s rest-gen/files/Docs/
```
It's caused by `$static$` in the `rest-gen/file/Docs/api-docs-{resource,all}.st`, I guess.
The `rest-gen/src/Rest/Gen/Docs.hs` doesn't seem to be preparing place holder for `$static$`.

**Comment by bergmark** [comment]
Fixed in `rest-gen-0.16.0.7`

**Comment by bergmark** [comment]
Oops, I mean `rest-gen-0.17.0.6`
